content/_index.md

```diff
@@ -1,5 +1,5 @@
 +++
-description = "Double Open is piloting a business providing OSS Review Toolkit (ORT) support for open source compliance needs of organizations. We want to solve the issue where tools and data are stuck in-house or behind proprietary licenses. Interested on more information or a pilot? Please provide your information below."
+description = "Double Open is piloting software composition analysis, or SCA, as a service, powered with OSS Review Toolkit (ORT). We want to solve the issue where tools and data are stuck in-house or behind proprietary licenses. Interested on more information or a pilot? Please provide your information below."
```

Comment:
Should we say "by" instead of "with"?

Also, in da0eccaeb634bf278f20dbf34b05ba0c18c2d313 I was going with a more general wording "Double Open is a company offering Open Source compliance solutions building on Open Source tools". I think it's fine to be more detailed here, but I still feel that we should align the wording a bit (in either direction). Also, is "piloting" still the right term?

Reply:
I would keep the message of piloting, since it gives us more leeway with respect to pilots, and realistically a lot of things develop all the time. We would move on in some months, I guess?
And "by" would be correct.